clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/jobs/local/JobExecutorLocal.groovy

```diff
@@ -138,8 +142,11 @@ class JobExecutorLocal implements JobExecutor {
           jobStatus.state = JobStatus.State.RUNNING
         }
 
-        if (logsContent) {
-          jobStatus.logsContent = logsContent
+        if (output) {
```

Comment:
Looks like this check isn't necessary since `output` is always assigned. Also, if `output` isn't assigned, why not just set `jobStatus.stdOut` to `output` which should be `null`?